ecs-agent/netlib/model/networkinterface/networkinterface.go

```diff
@@ -84,6 +84,10 @@ type NetworkInterface struct {
 	ipv4SubnetCIDRBlock    string
 	ipv6SubnetCIDRBlock    string
 
+	// Primary denotes whether the interface is responsible
+	// for handling default route within the netns it resides in.
+	Primary bool
```

Comment:
nit: naming -- if this designates the Default route, should this be `Default`?

Reply:
That makes sense. Will modify accordingly.